src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -307,14 +308,40 @@ extension DynamicTableViewController {
 		if dynamicTableViewModel.section(at: indexPath).isHidden(for: self) {
 			return UITableViewCell()
 		}
-
+		
 		let section = dynamicTableViewModel.section(at: indexPath)
 		let content = dynamicTableViewModel.cell(at: indexPath)
 
 		let cell = tableView.dequeueReusableCell(withIdentifier: content.cellReuseIdentifier, for: indexPath)
+		
+		
+		cell.removeBackground()
+		
+		switch section.background {
+		case .none:
+			cell.backgroundColor = .clear
```

Comment:
Can this maybe break some styling of existing cells?

Reply:
Indeed, i will remove that.